packages/react-components/src/components/ChatMessage/ChatMessageActionsFlyout.tsx

```diff
@@ -27,12 +31,19 @@ export interface ChatMessageActionFlyoutProps {
   onRemoveClick?: () => void;
   onDismiss: () => void;
   messageReadByCount?: number;
```

Comment:
Is this necessary if we have `messageReadByNames` below (i.e. does `messageReadByCount === messageReadByNames.length`?

Reply:
removed variable messageReadByCount